package.json

```diff
@@ -30,8 +31,12 @@
   },
   "homepage": "https://github.com/tsoding/emoteJAM#readme",
   "devDependencies": {
-    "@types/gif.js": "^0.2.1",
-    "@types/node": "^20.9.3",
-    "typescript": "^4.3.2"
+    "@types/gif.js": "^0.2.5",
+    "@types/node": "^22",
+    "typescript": "^5.5",
+    "vite": "^5.4.0"
+  },
+  "dependencies": {
+    "@dhdbstjr98/gif.js": "^1.0.2"
```

Comment:
removes the need to manually download the `gif.js` files (`gif.js, gif.map, gif.worker.js...`)